src/storage/mod.rs

```diff
@@ -22,10 +22,15 @@ mod config;
 #[cfg(feature = "storage-sqlite")]
 pub(crate) mod sqlite;
 
+#[cfg(feature = "storage-sqlite")]
+pub use sqlite::SqliteStorage;
```

Comment:
I was going to object to making these types public, as they have previously been accessed via `StorageConfig::into_storage`. However, that function returns `Box<dyn Storage>` and a boxed storage is no longer useful -- so I think `StorageConfig` itself is no longer useful.

All of which to say, I think this is fine, but let's also remove the StorageConfig type at the same time.